### PR TITLE
adds missing metadata declaration for normative interfaces

### PR DESCRIPTION
**Pull request by nyuuyn**
## Short Description
The interfaces definitions missed a metadata section which prevented working properly

### CODE REVIEW

File: interfacetypes/tosca.interfaces.node.lifecycle/Standard/InterfaceType.tosca [hunk 1]
```diff
@@ -2,6 +2,8 @@ tosca_definitions_version: tosca_simple_yaml_1_3
 interface_types:
   tosca.interfaces.node.lifecycle.Standard:
     derived_from: tosca.interfaces.Root
+	metadata:
```
Comment (v-yussupov) [inline interfacetypes/tosca.interfaces.node.lifecycle/Standard/InterfaceType.tosca]: @nyuuyn please, put the `metadata` on the same level as `derived_from`. See example:
image

File: interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca [hunk 2]
```diff
@@ -2,6 +2,8 @@ tosca_definitions_version: tosca_simple_yaml_1_3
 interface_types:
   tosca.interfaces.relationship.Configure:
     derived_from: tosca.interfaces.Root
+	 metadata:
```
Comment (v-yussupov) [inline interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca]: `metadata` on the same level as `derived_from`

File: interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca [hunk 3]
```diff
@@ -2,6 +2,8 @@ tosca_definitions_version: tosca_simple_yaml_1_3
...
+      targetNamespace: "tosca.interfaces"
```
Comment (v-yussupov) [inline interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca]: please, change to `tosca.interfaces.relationship`

File: interfacetypes/tosca.interfaces/Root/InterfaceType.tosca [hunk 4]
```diff
@@ -2,4 +2,6 @@ tosca_definitions_version: tosca_simple_yaml_1_3
 interface_types:
   tosca.interfaces.Root:
     derived_from: tosca.entity.Root
+	metadata:
```
Comment (v-yussupov) [inline interfacetypes/tosca.interfaces/Root/InterfaceType.tosca]: `metadata` on the same level as `derived_from`

**Review by v-yussupov — CHANGES_REQUESTED** [review]
thank you @nyuuyn, but some things need to be resolved before merging (see comments)

File: interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca (hunk 3 above)
Comment (nyuuyn) [reply interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca]: fixed

File: interfacetypes/tosca.interfaces.node.lifecycle/Standard/InterfaceType.tosca (hunk 1 above)
Comment (nyuuyn) [reply interfacetypes/tosca.interfaces.node.lifecycle/Standard/InterfaceType.tosca]: fixed

File: interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca (hunk 2 above)
Comment (nyuuyn) [reply interfacetypes/tosca.interfaces.relatioship/Configure/InterfaceType.tosca]: fixed

**Review by v-yussupov — APPROVED** [review]

**Pull request merged**